src/components/icons/Icon.jsx

```diff
@@ -106,7 +106,8 @@ export type IconTypeType =
   | 'triangle'
   | 'pi'
   | 'quote'
-  | 'spark';
+  | 'spark'
+  | 'rotate_90';
 
 export type IconColorType =
```

Comment:
u have to define it in `const TYPE` too.

then I suggest to run `yarn watch` and try to actually find the icon in the style-guide website 😉